src/courses/mappers/10.md

```diff
@@ -0,0 +1,45 @@
+---
+order: 10
+next: 11.md
+title: Mapper Preparations - The Art of Mapper Making
+author: Charles Hu
+---
+
+## The Art of Mapper Making
+
+Mapper creation is an inherently tricky process as every situation demands a unique and tailored solution. As a result, there is no one objective best template or process to follow as field translations will always be designed on a case-by-case basis. 
```

Comment:
> tricky

we don't want to scare people off.  nominally it's supposed to be easy to make a mapper.  i would say that it is a 'nuanced' process

Reply:
Addressed